ci/prod/build/build.sh

```
@@ -0,0 +1,20 @@
+
+if [ -n "$BUILD_SECRET" ]; then
+  BUILD_SECRET="--secret id=build_secret,src=$BUILD_SECRET"
+fi
+
+cleanup() {
+  docker container rm -f $BUILD_ID || true
+}
+
+trap cleanup EXIT INT TERM
+
+docker build -t $BUILD_IMAGE -f $BUILD_DOCKERFILE $BUILD_ARGS $BUILD_SECRET $BUILD_CONTEXT
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)